src/app-prisma/prisma/schema.prisma

```diff
@@ -0,0 +1,18 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
```

Comment:
I'd prefer to use variables, dbname, dbhost, etc..., as a result of that I can override the dbname from command line without the need to write the full url

Reply:
As well as, config variables must be loaded from .yaml files, in other words, using configService

Reply:
Left the usage at the constructor of `PrismaService`